src/test/java/edu/ucsb/cs156/example/controllers/PersonalSchedulesControllerTests.java

```diff
@@ -0,0 +1,588 @@
+        User u2 = User.builder().id(2L).build();
+        User u = currentUserService.getCurrentUser().getUser();
+
+        PersonalSchedule s1 = PersonalSchedule.builder().name("Schedule 1").description("Schedule 1").quarter("Quarter 1").user(u1).id(1L).build();
+        PersonalSchedule s2 = PersonalSchedule.builder().name("Schedule 2").description("Schedule 2").quarter("Quarter 2").user(u2).id(2L).build();
+        PersonalSchedule s3 = PersonalSchedule.builder().name("Schedule 3").description("Schedule 3").quarter("Quarter 3").user(u).id(3L).build();
+
+        ArrayList<PersonalSchedule> expected = new ArrayList<>();
+        expected.addAll(Arrays.asList(s1, s2, s3));
+
+        when(repo.findAll()).thenReturn(expected);
+
+        // act
+        MvcResult response = mockMvc.perform(get("/api/PersonalSchedules/admin/all"))
+                .andExpect(status().isOk()).andReturn();
+
+        // assert
+
+        verify(repo, times(1)).findAll();
+        String expectedJson = mapper.writeValueAsString(expected);
+        String responseString = response.getResponse().getContentAsString();
+        assertEquals(expectedJson, responseString);
+    }
+
+    // _________________________________________________________________________________
+    // Test for /api/PersonalSchedules/all
+
+    @WithMockUser(roles = { "USER" })
+    @Test
+    public void personal_schedules_all() throws Exception {
+
+        // arrange
+
+        User thisUser = currentUserService.getCurrentUser().getUser();
+
+        PersonalSchedule s1 = PersonalSchedule.builder().name("Schedule 1").description("Schedule 1").quarter("Quarter 1").user(thisUser).id(1L).build();
+        PersonalSchedule s2 = PersonalSchedule.builder().name("Schedule 2").description("Schedule 2").quarter("Quarter 2").user(thisUser).id(2L).build();
+        ArrayList<PersonalSchedule> expected = new ArrayList<>();
+        expected.addAll(Arrays.asList(s1, s2));
+        when(repo.findAllByUserId(thisUser.getId())).thenReturn(expected);
+
+        // act
+        MvcResult response = mockMvc.perform(get("/api/PersonalSchedules/all"))
+                .andExpect(status().isOk()).andReturn();
+
+        // assert
+
+        verify(repo, times(1)).findAllByUserId(eq(thisUser.getId()));
+        String expectedJson = mapper.writeValueAsString(expected);
+        String responseString = response.getResponse().getContentAsString();
+        assertEquals(expectedJson, responseString);
+    }
+
+    // _________________________________________________________________________________
+    // Tests for /api/PersonalSchedules
+
+    @WithMockUser(roles = { "USER" })
+    @Test
+    public void personal_schedule_exists__user() throws Exception {
+
+        // arrange
+
+        User u = currentUserService.getCurrentUser().getUser();
+        PersonalSchedule s1 = PersonalSchedule.builder().name("Schedule 1").description("Schedule 1").quarter("Quarter 1").user(u).id(7L).build();
```

Comment:
Same regarding quarter.  Please do a search and replace and make sure that all example values for quarter follow this convention